PendingReleaseNotes

```diff
@@ -182,3 +182,6 @@ Relevant tracker: https://tracker.ceph.com/issues/57090
   key/value set for a filesystem extended attributes.  It effectively replaces
   the old per-MDS `max_xattr_pairs_size` setting, which is now dropped.
   Relevant tracker: https://tracker.ceph.com/issues/55725
+* fs: two FS names can now be swapped, optionally along with their IDs, using
+ "fs swap" command. This commands also updates application tag on all the
+  pools of both the FSs and denies swapping if mirror is enabled.
```

Comment:
nit: Mention in separate line - File system mirror should be disabled for the swapped file systems.

Reply:
Done.